charts/cockroachdb/19.1.5/app-readme.md

```diff
@@ -0,0 +1,381 @@
+# CockroachDB Helm Chart
```

Comment:
since `app-readme.md` is a file that provides descriptive text in the chart’s UI header.  so we don't need a full doc copy of README.